### PR TITLE
chore: Add option to build some qtox deps as static lib.

### PR DESCRIPTION
**Pull request by iphydf**
This change is [Reviewable](https://reviewable.io/reviews/TokTok/dockerfiles/185)

### CODE REVIEW

File: qtox/build_qrencode.sh [hunk 1]
```diff
@@ -17,10 +17,10 @@ parse_arch --dep "qrencode" --supported "win32 win64 macos macos-x86_64 macos-ar
 cmake . \
   -DCMAKE_BUILD_TYPE=Release \
   -DCMAKE_INSTALL_PREFIX="$DEP_PREFIX" \
-  "-DCMAKE_OSX_DEPLOYMENT_TARGET=$MACOS_MINIMUM_SUPPORTED_VERSION" \
+  -DCMAKE_OSX_DEPLOYMENT_TARGET="$MACOS_MINIMUM_SUPPORTED_VERSION" \
   "$CMAKE_TOOLCHAIN_FILE" \
   -DWITH_TOOLS=OFF \
-  -DBUILD_SHARED_LIBS=ON
+  -DBUILD_SHARED_LIBS=OFF
```
Comment (robinlinden) [inline qtox/build_qrencode.sh:23]: Should this one not be toggleable?

File: qtox/build_sqlcipher.sh [hunk 2]
```diff
@@ -19,26 +19,26 @@ parse_arch --dep "sqlcipher" --supported "win32 win64 macos macos-x86_64 macos-a
...
+else
+  LIBS=''
 fi
 
 ./configure "$HOST_OPTION" \
-  "--prefix=$DEP_PREFIX" \
-  --enable-shared \
-  --disable-static \
+  --prefix="$DEP_PREFIX" \
+  --enable-static \
```
Comment (robinlinden) [inline qtox/build_sqlcipher.sh:33]: No choice between shared and static here?

**Review by robinlinden — APPROVED** [review]

File: qtox/build_qrencode.sh (hunk 1 above)
Comment (iphydf) [reply qtox/build_qrencode.sh]: Yep, done.

File: qtox/build_sqlcipher.sh (hunk 2 above)
Comment (iphydf) [reply qtox/build_sqlcipher.sh]: Thanks, missed this one.

**Pull request merged**